packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -425,15 +425,15 @@ function FontLibraryProvider( { children } ) {
 
 		const isFaceActivated = isFontActivated(
 			font.slug,
-			face.fontStyle,
-			face.fontWeight,
+			face?.fontStyle,
+			face?.fontWeight,
```

Comment:
This doesn't seem to be right because `fontStyle` and `fontWeight` are used to determine if the face is active or not so they are always needed.